onager/backends/slurm.py

```diff
@@ -75,3 +112,12 @@ def get_job_list(self, args):
         wrapper_file = self.save_wrapper_script(wrapper_script, args.jobname)
         base_cmd += "{}".format(wrapper_file)
         return [base_cmd]
+
+    def launch(self, jobs, args, other_args):
+        task_ids = expand_ids(args.tasklist)
+        task_id_strs = ','.join(map(str, task_ids))
```

Comment:
The feature works by taking all the ids you care about, and grouping them into jobs. Much easier to do that when they're expanded. I may be able to only do this part in _get_body, I don't remember if I tried that.